webapp-client/src/component/CategoryList.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+// TODO: Add support to highlight active category.
+export default class CategoryList extends React.Component {
+  // this.props.categories: a list of category ids fetched from db
+  // this.props.setContentPane: sets the current category to be displayed
+  render() {
+    return (
+      <React.Fragment>
```

Comment:
Can you maintain consistency between the imports so that it is always "React.Fragment" or "Fragment" throughout the app? (See app.js)

Reply:
We changed app to be consistent with this and we will be using React.Fragment from now on:))